### PR TITLE
1307 multiple log files

### LINKED ISSUE #1307: Some issue with logging still
@georgeDaskalakis Here is what I did
Get the latest of qualificationRunner and PKSim

Clone repo Efavirenz-Model
https://github.com/Open-Systems-Pharmacology/Efavirenz-Model

In the QualificationRunner folder, start the runner with the following parameters

`QualificationRunner.exe -i "C:\projects\Efavirenz-Model\Evaluation\Input\evaluation_plan.json" -o "C:\tests\10.0\Efavirenz\output1" -f -p "C:\projects\PK-Sim\src\PKSim\bin\Debug\net472" -l "C:\tests\10.0\Efavirenz\log.txt" --norun --logLevel Debug`

Observe that there are many log files everywhere. 

3 for the main log
image

And 3 for each instance of PKSIm started (I started two). Can it be that there is something global going on here?
image
and

image

### PR DESCRIPTION
**Pull request by georgeDaskalakis**
closes #1307

### CODE REVIEW

File: src/OSPSuite.Core/Qualification/QualifcationConfiguration.cs [hunk 1]
```diff
@@ -36,6 +36,12 @@ public class QualifcationConfiguration : IValidatable
       /// </summary>
       public string MappingFile { get; set; }
 
+      /// <summary>
+      ///    Path of log file that will be created for the qualification runner.
+      ///    and will include all entries from all qualification processes run.
+      /// </summary>
+      public string QRSharedLogPath { get; set; }
```
Comment (msevestre) [inline src/OSPSuite.Core/Qualification/QualifcationConfiguration.cs:43]: I don't understand why we need this? Log files are passed as parameter to the cli...

File: src/OSPSuite.Infrastructure/Services/LoggingBuilderExtensions.cs [hunk 2]
```diff
@@ -30,18 +37,18 @@ public static ILoggingBuilder AddFile(this ILoggingBuilder builder, string logFi
...
-         //1073741824 is as far as I understand the maximum possible fileSizeLimitBytes
-         builder.AddSerilog(
-           new LoggerConfiguration()
-              .MinimumLevel.ControlledBy(new LoggingLevelSwitch { MinimumLevel = serilogLevel })
-              .WriteTo.File( logFileFullPath, fileSizeLimitBytes: 1073741824, rollOnFileSizeLimit: true, restrictedToMinimumLevel: serilogLevel, outputTemplate: "[{Timestamp:yyyy-MM-dd HH:mm:ss.fff} {SourceContext:l} {Level:u}] {Message:l} {NewLine:l} {Exception}")
-             .CreateLogger()
-         );
+         var loggerConfiguration = new LoggerConfiguration().MinimumLevel.ControlledBy(new LoggingLevelSwitch { MinimumLevel = serilogLevel });
+
+         foreach (var logFile in logFileFullPaths)
```
Comment (msevestre) [inline src/OSPSuite.Infrastructure/Services/LoggingBuilderExtensions.cs:42]: Looks like you removed the comment. I would keep it as it explains why those weird values are set

Comment (georgeDaskalakis) [reply src/OSPSuite.Infrastructure/Services/LoggingBuilderExtensions.cs:42]: you are right, I copied it to paste it underneath and then totally forgot. corrected

File: src/OSPSuite.Core/Qualification/QualifcationConfiguration.cs (hunk 1 above)
Comment (georgeDaskalakis) [reply src/OSPSuite.Core/Qualification/QualifcationConfiguration.cs:43]: you are right, I was copying it here without reason. removed

**Comment by msevestre** [comment]
awesome

**Pull request merged**